triagebot.toml

```diff
@@ -738,6 +739,9 @@ ast_lowering = [
     "@estebank",
     "@spastorino",
 ]
+debuginfo = [
+    "@michaelwoerister",
```

Comment:
```suggestion
    "@michaelwoerister",
    "@davidtwco"
```

Reply:
Maybe @wesleywiser as well, having been active in wg-debugging?

Reply:
Let's ask him 🙂 He certainly knows more about the MIR-side of debuginfo than I do (although I'd like to learn).